vignettes/charlatan.Rmd

```diff
@@ -169,6 +169,33 @@ ch_credit_card_security_code()
 ch_credit_card_security_code(10)
 ```
 
+## Missing data 
+`charlatan` makes it very easy to generate fake data with missing entries. First, you need to run `MissingDataProvider()` and then make an appropriate `make_missing()` call specifying the data type to be generated. This method picks a random number (`N`) of slots in the input the `make_missing` vector and then picks `N` random positions that will be replaced with NA matching the input class.
```

Comment:
> of slots in the input the `make_missing` vector

"of slots in the input `make_missing` vector"

Reply:
now done in d3adc9972ea7850d2789070d844033d743cf53a6